tests/acceptance/features/serviceAvailability/serviceAvailabilityCheck.feature

```diff
@@ -0,0 +1,133 @@
+Feature: service health check
+
+
+  Scenario: check default services health
+    When a user requests these endpoints:
+      | endpoint                                        | service            | comment |
+      # | %base_url_without_scheme_and_port%:9197/healthz | activitylog        | #get 500 |
```

Comment:
we should uncomment it and add it to expected failure with an issue. there should be an existing issue somewhere